### PR TITLE
tldrl: add alias to tldr-lint

### LINKED ISSUE #5444: Page request: tldr-lint
```bash
$ tldr tldr-lint
`tldr-lint` documentation is not available. Consider contributing Pull Request to https://github.com/tldr-pages/tldr
```

### PR DESCRIPTION
**Pull request by bl-ue**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Closes #5444

### CODE REVIEW

**Comment by bl-ue** [comment]
cc @Yutyo: once this is merged, could we get a quick Turkish translation?  It would be useful in the future, because the format of alias pages is simple enough for users to copy/paste for other commands and just change the references to the other command.  Get it?

File: pages/common/tldr-lint.md
Comment (MasterOdin) [inline pages/common/tldr-lint.md:1]: `s/tldrl/tldr-lint` through this file?

Comment (bl-ue) [reply pages/common/tldr-lint.md:1]: Sorry about that, just another copy/paste error 🙂

**Review by owenvoke — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Comment by Yutyo** [comment]
If its all OK, can I merge this?

**Comment by bl-ue** [comment]
@Yutyo, yes, you can merge it.  But first approve it.  As a matter of habit, always approve PRs before merging (even if they have enough approvals already.)

**Comment by bl-ue** [comment]
Also @Yutyo, please clear the extended commit message (`Ctrl+A` + `Backspace`) before merging.

We squash so that all commits may become clean, but here you can see I made a messy commit (`e`):
>```
>* tldrl: rename to tldr-lint
>
>* tldrl: add page
>
>* tldr-lint: s/tldrl/tldr-lint/
>
>* tldr-lint: prefer long options
>
>* e
>```

Not a big deal at all, just a reminder.  We're going to add that to the maintainers guide at some point...

**Comment by marchersimon** [comment]
@bl-ue I think it's already [there](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md#ii-handling-prs) in the second paragraph from the bottom

**Comment by bl-ue** [comment]
Yeah, I've never noticed that :)

>(Don't forget to clean up the body of the squashed commit message.)

Hmm...maybe that stuff needs a more promenent place in the guidelines, as it gets missed frequently, I'd say.

**Pull request merged**
